packages/b/boost/xmake.lua

```diff
@@ -302,9 +307,28 @@ package("boost")
                 cxxflags = cxxflags .. " -fPIC"
             end
         end
+        if package:runtimes("c++_shared", "c++_static") then
```

Comment:
`if package.has_runtime and package:has_runtime("c++_shared", "c++_static")`

This interface is only available in dev.